apps/dynatrace/aat/base/dynakube.yaml

```
@@ -4,8 +4,14 @@ metadata:
   name: dynakube
   namespace: dynatrace
 spec:
-  apiUrl: https://yrk32651.live.dynatrace.com/api
+  apiUrl: 'https://yrk32651.live.dynatrace.com/api'
   oneAgent:
     cloudNativeFullStack:
       args:
-        - --set-host-group=AAT_CFT
+        - '--set-host-group=AAT_CFT'
+  namespaceSelector:
+    matchExpressions:
+      - key: helm.toolkit.fluxcd.io/namespace
```

I would use the actual namespace name, copied from https://github.com/hmcts/sds-flux-config/blob/ad1da12cbc9de4a797a476220f4f71505f832d36/apps/dynatrace/dynakube.yaml#L68-L73
```suggestion
      - key: kubernetes.io/metadata.name
```

---

Yeah agree with that, updated